Fmaraev/Hometask_5/Task_3.py

```diff
@@ -0,0 +1,17 @@
+"""Задача Дан список стран и городов каждой страны. Затем даны названия
+городов. Для каждого города укажите, в какой стране он находится."""
+
```

Comment:
please separate logical blocks with lines
please reduce number of magic variables (i m n j )